opentabs.py

```diff
@@ -199,34 +182,11 @@ def create_file_panel_item(self, some_content):
       return sublime.QuickPanelItem(buffer_content.tab_name, "", "unsaved", sublime.KIND_NAVIGATION)
 
   def when_file_selected(self, index):
+    window = self.window
     user_selection = self.selected_index if index == -1 else index
     if user_selection != -1 and self.tracked_views and len(self.tracked_views) > user_selection:
       some_content = self.tracked_views[user_selection]
-      if type(some_content) == FileContents:
-        self.find_tab_by_filename(some_content)
-      else:
-        self.find_tab_by_name(some_content)
-
-
-  def find_tab_by_filename(self, file_content):
-    window = self.window
-    view = window.find_open_file(file_content.file_name)
-    if view:
-      window.focus_view(view)
-
-  def find_tab_by_name(self, tab_content):
-    window = self.window
-    view = self.find_view_by_tab_name(tab_content.tab_name)
-    if view:
-      window.focus_view(view)
-
-
-  def find_view_by_tab_name(self, tab_name):
-    for view in self.buffers:
-      if not view.file_name() and view.name() == tab_name:
-        return view
-
-    return None
+      window.focus_view(some_content)
```

Comment:
How does this work? `tracked_views` has `FileContents` or `BufferContents` not a view? Does something convert it into the view?